### ISSUE TITLE
You are not gonna read it!

### ISSUE THREAD

**Issue by Seidkhan-Yusupov** [KalobTaulien/git-essentials#161]
Hey ma bois I think there is no any issues to mention so, I think we better have fun exploring github!

**Comment by iwsugata** [comment]
Yeah we have not yet into real issue..just try every thing possible :))
